### PR TITLE
Hybrid model: Make dataframe column names configurable. I also fixed the example in the docstring

### PR DESCRIPTION
**Pull request by sognetic**
Pretty much what it says in the title.

### CODE REVIEW

File: eFFORT/hybrid/hybrid_binning.json [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
\ No newline at end of file
+    "MX": {
+        "column_name": "mX",
+        "bins": [0.0, 1.4, 1.6, 1.8, 2.0, 2.5, 3.0, 3.5]
+  },
+  "ELB":{
+        "column_name": "El_B",
+        "bins": [0.0, 0.5, 1.0, 1.25, 1.5, 1.75, 2.0, 2.25, 3.0]
+    },
+    "Q2": {
```
Comment (MarkusPrim) [inline eFFORT/hybrid/hybrid_binning.json:10]: Indentation looks a bit off

File: eFFORT/hybrid/hybrid_model.py [hunk 2]
```diff
@@ -69,7 +77,7 @@ def calculate_weight(self, x: numpy.array, hybrid_weights: numpy.histogramdd) ->
         syntax, assuming you stored the result of :func:`~hybrid.Hybrid.generate_hybrid_weights` in the variable
         hybrid_weights, and your data is stored in the pandas.DataFrame df:
 
-            >>> # df['hybrid_weight'] = calculate_weight(df[['El_B', 'q2', 'mX']], hybrid_weights)
+            >>> # df['hybrid_weight'] = calculate_weight(df[['El_B', 'q2', 'mX']].values, hybrid_weights)
```
Comment (meliache) [inline eFFORT/hybrid/hybrid_model.py:80]: Not part of your changes, but is the hash after the prompt there for a reason? I see no reason why the example should be a commented out line?

Nice that you added the `.values` :+1: 

Btw, I didn't know that sphinx renders interactive prompts automatically if you include the prompt, I just found out when I checked if the `>>>` would work, I had only used code blocks before.

Comment (MarkusPrim) [reply eFFORT/hybrid/hybrid_model.py:80]: Yes the hash is there in case you run the doctest, as this would otherwise fail.

**Comment by sognetic** [comment]
Can this be merged? Indentation has been fixed

**Pull request merged**